enterprise/static/enterprise/enterprise_landing_page.css

```diff
@@ -0,0 +1,503 @@
+* {
```

Comment:
It's copy and pasted from the other CSS files because they will be sharing style a lot -- ignore for now.